0001-kaa-protocol/README.md

```diff
@@ -97,9 +97,9 @@ Having a predefined prefix helps routing all KP-related traffic through MQTT bro
 
 `<extension_instance_name>` is a name that uniquely identifies an extension instance within the application.
 
-Extension-specific resource path part for endpoint-aware extensions MUST start with the endpoint token.
+Extension-specific resource path part for endpoint-aware extensions MUST start with the endpoint identifier ("endpoint token").
```

Comment:
Upd: Not true, it is defined in the Security section that still requires review.